src/components/grid-aware/ThreeParagraphBlock/ThreeParagraphBlock.js

```diff
@@ -42,8 +43,8 @@ const CTABlock = ({ title, buttons }) => (
     </div>
     <div className={s.ctaButtonRow}>
       {buttons.map((button) => (
-        <div className={s.ctaButtonRowItem}>
-          <Button key={button.text}>{button.text}</Button>
+        <div className={s.ctaButtonRowItem} key={button.text}>
```

Comment:
![Screen Shot 2020-10-19 at 7 51 58 PM](https://user-images.githubusercontent.com/30575095/96535150-1092d100-1246-11eb-89d6-6d931a4ddde5.png)

I refactored the parent element of the body in the map function to make this warning go away.